kalite/distributed/management/commands/retrievecontentpack.py

```diff
@@ -0,0 +1,102 @@
+"""
+Management command for downloading a language pack and extracting the
+contents to their correct locations.
+
+Usage:
+  contentpackretrieve download <lang>
+  contentpackretrieve retrieve <lang> <packpath>
+  contentpackretrieve -h | --help
+
+"""
+import json
+import os
+import urllib
+import tempfile
+import shutil
+import zipfile
+
+from django.core.management.base import BaseCommand
+
+from fle_utils.general import ensure_dir
+
+from kalite.i18n.base import lcode_to_django_lang, get_po_filepath, get_locale_path
+from kalite.topic_tools import settings
+
+from kalite.version import SHORTVERSION
+
+CONTENT_PACK_URL_TEMPLATE = ("http://pantry.learningequality.org/downloads" +
+                             "/ka-lite/{version}/content/contentpacks/{code}.zip")
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+
+        # TODO: change the codepath depending on whether they want to download
+        # or simply extract a content pack.
+
+        # parse out the options and raise errors if necessary
+        lang = args[0]
+
+        with tempfile.NamedTemporaryFile() as f:
+            zf = download_content_pack(f, lang)
+
+            extract_catalog_files(zf, lang)
+            extract_content_db(zf, lang)
+            extract_content_pack_metadata(zf, lang)
+
+
+def extract_content_pack_metadata(zf, lang):
+    # stub for now, until we implement metadata creation on the maker side.
+    metadata_path = os.path.join(get_locale_path(lang), "{lang}_metadata.json".format(lang=lang))
+    barebones_metadata = {
+        "code": lang,
+        'software_version': SHORTVERSION,
+        'language_pack_version': 1,
+        'percent_translated': 100,
+        'subtitle_count': 0,
+        "name": "DEBUG",
+        'native_name': 'DEBUG',
+    }
+
+    with open(metadata_path, "wb") as f:
+        json.dump(barebones_metadata, f)
+
+
+def download_content_pack(fobj, lang):
+    url = CONTENT_PACK_URL_TEMPLATE.format(
+        version=SHORTVERSION,
+        code=lang,
+    )
+
+    urllib.urlretrieve(url, filename=fobj.name)
```

Comment:
Why this rather than `requests.get(stream=True)`? Because the bundled version of `requests` doesn't support `stream`.